doc/association-rules.rst

```diff
@@ -0,0 +1,74 @@
+Association Rules
+=================
+
+.. figure:: icons/association-rules.png
+
+Induction of association rules.
+
+Signals
+-------
+
+**Inputs**:
+
+-  **Data**
+
+Data set
+
+**Outputs**:
+
+-  **Matching Data**
+
+Data instances matching the criteria.
+
+Description
+-----------
+
+This widget runs several algorithms for induction of association rules: the
+original Agrawal's algorithm for sparse data sets, a modified version by Zupan
+and Demšar, which is more suitable for the usual machine learning data, and, finally,
+an algorithm which induces classification rules where the right-hand side of the
+rule is always a class attribute.
```

Comment:
Sorry, what? :smiley: This add-on contains only the [documented algorithms](https://orange3-associate.readthedocs.io/en/latest/).

The algorithm generates classification rules in the exact same way as those with arbitrary consequent, except that it filters out the rules where the consequent does not match one of the class' values.